pkg/installmanager/installmanager.go

```diff
@@ -1622,19 +1623,21 @@ func waitForProvisioningStage(m *InstallManager) error {
 	waitContext, cancel := context.WithTimeout(context.Background(), provisioningTransitionTimeout)
 	defer cancel()
 
+	scheme := testfake.GetScheme()
+
 	config, err := rest.InClusterConfig()
 	if err != nil {
 		return errors.Wrap(err, "could not get in-cluster REST config")
 	}
-	gvk, err := apiutil.GVKForObject(&hivev1.ClusterProvision{}, scheme.Scheme)
+	gvk, err := apiutil.GVKForObject(&hivev1.ClusterProvision{}, scheme)
 	if err != nil {
 		return errors.Wrap(err, "could not get the GVK for clusterprovisions")
 	}
 	hc, err := rest.HTTPClientFor(config)
 	if err != nil {
 		return errors.Wrap(err, "could not generate http client for config")
 	}
-	restClient, err := apiutil.RESTClientForGVK(gvk, false, config, scheme.Codecs, hc)
+	restClient, err := apiutil.RESTClientForGVK(gvk, false, config, serializer.NewCodecFactory(scheme), hc)
```

Comment:
Yes -- or you can do this next to your `testfake.GetScheme()`, making a global-to-hive codec factory.